nltools/utils.py

```diff
@@ -287,15 +287,21 @@ def check_square_numpy_matrix(data):
     return data
 
 
-def check_brain_data(data):
+def check_brain_data(data, mask=None):
     '''Check if data is a Brain_Data Instance.'''
     from nltools.data import Brain_Data
 
     if not isinstance(data, Brain_Data):
         if isinstance(data, nib.Nifti1Image):
-            data = Brain_Data(data)
+            if mask is None:
```

Comment:
This can be simplified by removing the if statement because `Brain_Data` can already handle a `None` value for mask, so replace with:  

`data = Brain_Data(data, mask=mask)`